opendevin/server/session/manager.py

```diff
@@ -48,13 +48,15 @@ async def _cleanup_sessions(self):
             for sid, session in list(self._sessions.items()):
                 # if session inactive for a long time, remove it
                 if (
-                    not session.is_alive
-                    and current_time - session.last_active_ts > self.session_timeout
+                        not session.is_alive
+                        and current_time - session.last_active_ts > self.session_timeout
                 ):
                     session_ids_to_remove.append(sid)
 
             for sid in session_ids_to_remove:
-                del self._sessions[sid]
-                logger.info(f'Session {sid} has been removed due to inactivity.')
+                session: Optional[Session] = self._sessions.pop(sid, None)
```

Comment:
linting failed due to reuse of variable name `session`.

I am surprised you were able to commit this - pre-commit hook should be able to detect this. Did you disable/skip the pre-commit hook?

Reply:
Well, I'm a beginner to Python, and the pre-commit hook is disabled since there's conflicts with my company pre -hook. I'll figure out a way to solve this later. Thanks for pointing out the problem here. @li-boxuan

Reply:
I see. You could also run `make lint`.